app/pods/publications/publication/documents/template.hbs

```diff
@@ -60,13 +58,14 @@
   {{#if this.showFilterPanel}}
     <div class="auk-u-mb-2">
       <Publications::Publication::Documents::DocumentsFilter
+        @types={{hash documentTypes=this.documentTypes}}
         @filter={{this.filter}} @onChange={{this.onPerformFilter}} />
     </div>
   {{/if}}
 
-  {{#if (await this.initialDocumentLoad)}}
+  {{#if this.isLoaded}}
     <div class="auk-u-mb-3">
-      {{#if (gt (await this.filteredSortedPieces.length) 0 )}}
```

Comment:
Misschien stond dit hier voor een reden, maar volgens mij was en is filteredSortedPieces een simpele array, en dus niet thenable.